Tests/LibWeb/Text/input/css/dir-pseudo-on-input-element.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<style>
+    .test {
+        font-size: 12px;
+    }
+</style>
+<script src="../include.js"></script>
+<script>
+    test(() => {
+        function testSelectorMatch(input, selector) {
+            println(`Input matches ${selector}: ${input.matches(selector)}`);
+        }
+
+        const input = document.createElement("input");
+        input.type = "text";
+        input.value = "Well hello friends!"
+        testSelectorMatch(input, ":dir(ltr)");
+        testSelectorMatch(input, ":dir(rtl)");
+
+        testSelectorMatch(input, ":dir(ltr)");
+        testSelectorMatch(input, ":dir(rtl)");
```

Comment:
These two are a duplicate of the ones above, are they meant to test something else?

Reply:
Not sure what happened there. I've updated the test with what was supposed to be there.